py-polars/polars/dataframe/frame.py

```diff
@@ -4319,11 +4319,10 @@ def describe(
         -----
         The median is included by default as the 50% percentile.
 
-        Warning
-        -------
-        We will never guarantee the output of describe to be stable.
-        It will show statistics that we deem informative and may
-        be updated in the future.
+        .. warning::
+            We will never guarantee the output of describe to be stable.
+            It will show statistics that we deem informative and may
+            be updated in the future.
```

Comment:
technically there isn't a "Warning" section, and it's currently flagged in the build:
```
updating environment: [new config] 1233 added, 0 changed, 0 removed
/home/marcogorelli/polars-dev/.venv/lib/python3.10/site-packages/numpydoc/docscrape.py:455: UserWarning: Unknown section Warning in the docstring of describe in /home/marcogorelli/polars-dev/py-polars/polars/dataframe/frame.py.
  warn(msg)
/home/marcogorelli/polars-dev/.venv/lib/python3.10/site-packages/numpydoc/docscrape.py:455: UserWarning: Unknown section Warning in the docstring of describe in /home/marcogorelli/polars-dev/py-polars/polars/dataframe/frame.py.
  warn(msg)
/home/marcogorelli/polars-dev/.venv/lib/python3.10/site-packages/numpydoc/docscrape.py:455: UserWarning: Unknown section Warning in the docstring of describe in /home/marcogorelli/polars-dev/py-polars/polars/dataframe/frame.py.
  warn(msg)
```

There is a "Warnings" section, but that's meant to be for the warnings which a function may emit

So I think this can just be a warning directive